gatsby/src/components/Subline.js

```diff
@@ -1,11 +1,11 @@
-import styled from 'styled-components'
+import styled from 'styled-components';
 
 const Subline = styled.div`
   font-size: ${props => props.theme.fontSize.small};
   color: ${props => props.theme.colors.grey.light};
   ${props => props.sectionTitle && 'margin-top: -3rem'};
-  ${props => props.sectionTitle && 'margin-bottom: 4rem'};
+  ${props => (props.sectionTitle && 'margin-bottom: 4rem') || 'margin-bottom: 1em;'};
```

Comment:
Add some default margin to the bottom so the subline(where the date and author is) is not squished up against the blog post content.